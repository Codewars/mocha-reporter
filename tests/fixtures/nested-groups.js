const assert = require("assert");

describe("group 1", () => {
  describe("group 1 1", () => {
    it("test 1", () => {
      assert.strictEqual(1, 1);
    });
  });
});
