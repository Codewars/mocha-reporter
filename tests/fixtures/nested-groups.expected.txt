
<DESCRIBE::>group 1

<DESCRIBE::>group 1 1

<IT::>test 1

<PASSED::>Test Passed

<COMPLETEDIN::>0

<COMPLETEDIN::>0

<COMPLETEDIN::>0
